src/common/utility.ts

```diff
@@ -415,15 +415,18 @@ export const updateURLAsPerUserLanguage = () => {
     }
     const user_language = Cookies.get('user_language') || 'en'
 
-    const language = has_language_in_url ? first_path : user_language
-
     if (!has_language_in_url && user_language === 'en') return
     if (first_path === user_language) return
+    if (first_path !== user_language) {
+        Cookies.remove('user_language')
+        setCookiesWithDomain('user_language', first_path)
+        return
+    }
```

Comment:
why are we using two if statements? can we do it in if else

Reply:
we can't use `else return` in this case. if we use, the codes below this condition will be unreachable